src/Impostor.Server/Net/Inner/Objects/GameManager/Logic/GameLogicComponent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Impostor.Api.Net.Inner;
+using Impostor.Api.Net.Inner.Objects;
+
+namespace Impostor.Server.Net.Inner.Objects.GameManager.Logic;
+
+internal abstract class GameLogicComponent
+{
+    public virtual void OnPlayerDisconnect(IInnerPlayerControl pc)
+    {
+    }
+
+    public virtual void HandleRPC(RpcCalls callId, IMessageReader reader)
```

Comment:
`HandleRpcAsync` or remove altogether until its actually needed?

Reply:
I just copied what the base game did, maybe keep it and put a `throw NotImplementedException()` so we know when it is used in the future.